tests/models/store.py

```diff
@@ -1764,3 +1764,22 @@ def test_update_resurrect(store_po, test_fs):
     assert units.count() == len(obsolete_ids)
     for unit in units:
         assert not unit.isobsolete()
+
+
+@pytest.mark.django_db
+def test_store_comment_update(store0, member):
+    ttk = store0.deserialize(store0.serialize())
+    fileunit = ttk.units[-1]
+    fileunit.removenotes()
+    fileunit.addnote("A new comment")
+    store0.update(
+        ttk, store_revision=store0.data.max_unit_revision + 1,
+        user=member
+    )
+    assert ttk.units[-1].getnotes("translator") == "A new comment"
+    unit = store0.units.last()
+    assert unit.translator_comment == "A new comment"
+    assert unit.commented_by == member
+    sub = unit.submission_set.all().order_by("id").last()
```

Comment:
lets use the pattern used elsewhere to ensure that this is the only submission created - ie capturing last_unit_pk before change